probeinterface/io.py

```diff
@@ -139,6 +139,63 @@ def read_prb(file):
     return probegroup
 
 
+def read_maxwell(file, well_name='well000', rec_name='rec0000'):
+    """
+    Read a maxwell file and return a Probe object.
+
+    Since Maxwell do not handle electrode shape then circle of 5um are put.
+    Same for electrode shape a dummy tip is put.
+
+    Maxwell format do not contain any information about the channel of the probe
+    Only the channel index on device is given.
+
+    """
+
+    file = Path(file).absolute()
+    assert file.is_file()
+
+    try:
+        import h5py
+    except ImportError as error:
+        print(error.__class__.__name__ + ": " + error.message)
+
+
+    my_file = h5py.File(file, mode='r')
+
+    if 'mapping' in my_file.keys():
+        mapping = my_file['mapping'][:]
+        nb_channels = my_file['sig'].shape[0]
+    else:
+        mapping = my_file['wells'][well_name][rec_name]['settings']['mapping'][:]
+        nb_channels = my_file['sig'].shape[0]
+
+    prb = {'channel_groups' : {1 : {}}}
+
+    channels = list(mapping['channel'])
+    x_pos = list(mapping['x'])
+    y_pos = list(mapping['y'])
+    geometry = {}
+    for c, x, y in zip(channels, x_pos, y_pos):
+        geometry[c] = [x, y]
+
+    my_file.close()
+
+    prb['channel_groups'][1]['geometry'] = geometry
+    prb['channel_groups'][1]['channels'] = channels
+
+    probe = Probe(ndim=2, si_units='um')
+
+    chans = np.array(prb['channel_groups'][1]['channels'], dtype='int64')
+    positions = np.array([prb['channel_groups'][1]['geometry'][c] for c in chans], dtype='float64')
+
+    probe.set_electrodes(positions=positions, shapes='rect', shape_params={'width': 5.45, 'height' : 9.3})
+    probe.create_auto_shape(probe_type='rect')
```

Comment:
Maxwell chips always have the same shape, independent of the config. I would set it manually:

```suggestion
    probe.set_planar_contour(([-12.5, -12.5], [3845, -12.5], [3845, 2095], [-12.5, 2095]))
```